plugins/keycloak-backend/src/lib/read.test.ts

```diff
@@ -0,0 +1,19 @@
+import KcAdminClient from '@keycloak/keycloak-admin-client';
+
+import { KeycloakAdminClientMock } from '../../__fixtures__/helpers';
+import { KeycloakProviderConfig } from './config';
+import { readKeycloakRealm } from './read';
+
+const config = {
+  realm: 'myrealm',
+} as unknown as KeycloakProviderConfig;
```

Comment:
nit: do we need to case as unknown? Can directly do `as KeycloakProviderConfig` or better to add required type with mock

```
const config: KeycloakProviderConfig = {
  realm: 'myrealm',
  id: 'mock_id',
  baseUrl: 'http://mock-url',
};
```

Reply:
Sure, I'll update the object with the required types since there are only 3 of them.